w3af/plugins/crawl/open_api.py

```diff
@@ -314,6 +366,18 @@ def get_options(self):
         o = opt_factory('header_auth', self._header_auth, d, HEADER, help=h)
         ol.add(o)
 
+        d = 'Disable Open API spec validation'
+        h = 'By default, the plugin validates Open API specification before extracting endpoints.'
+        o = opt_factory('no_spec_validation', self._no_spec_validation, d, BOOL, help=h)
+        ol.add(o)
+
+        d = 'Path to Open API specification'
+        h = ('By default, the plugin looks for the API specification on the target,',
+             ' but sometimes applications do not provide an API specification. ',
+             ' Set this parameter to specify a local path to the API specification.')
+        o = opt_factory('custom_spec_location', self._custom_spec_location, d, STRING, help=h)
```

Comment:
Change this one and we're A-OK for merging :+1:

Reply:
Good point! I'll update it, thanks!

Reply:
I updated the type to be `INPUT_FILE`. Now if the file doesn't exist, the plugin reports it during configuration. Then, crawling/testing go as usual, but most likely no API will be tested because no API spec provided. It looks to me as an expected behavior, but it can hide configuration issues if w3af is run with a script (`-s` option). Does w3af have an option to exit with a non-zero code if a configuration step failed? It would help to identify configuration issues (it's probably out of scope in this PR).